services/template-repository/app/models/schemas/flowTemplate.js

```diff
@@ -77,6 +77,7 @@ const flowTemplate = new Schema({
   status: { type: String, default: 'draft' },
   cron: { type: String },
   flowSettings: {},
+  fromTemplate: { type: mongoose.Types.ObjectId },
```

Comment:
By adding "fromTemplate" field in the template, will we have problems passing the "fromTemplate" field on to the flow from the original template rather than the template that the flow is actually generated from? i.e. since the names are the same, is there an issue in the flow-repository of overwriting the id with this new id?

Reply:
Yes, sorry. We want to be able to copy templates from one user to another. I thought about this field to be able to follow from which one it was copied. But, yes, that’s a good point. What do you think about "copiedFrom"?

Reply:
That is also fine, and removes the chance of any copy errors (I wasn't sure it would happen, just wanted to make sure that you had checked 😄 ). Also happy with names like "sourceTemplate".

Reply:
Yep, I like it more as well!